extensions/mssql/src/features.ts

```diff
@@ -921,3 +921,165 @@ export class SqlAssessmentServicesFeature extends SqlOpsFeature<undefined> {
 
 
 }
+
+export class ProfilerFeature extends SqlOpsFeature<undefined> {
+	private static readonly messagesTypes: RPCMessageType[] = [
+		contracts.StartProfilingRequest.type,
+		contracts.StopProfilingRequest.type,
+		contracts.ProfilerEventsAvailableNotification.type
+	];
+
+	constructor(client: SqlOpsDataClient) {
+		super(client, ProfilerFeature.messagesTypes);
+	}
+
+	public fillClientCapabilities(capabilities: ClientCapabilities): void {
+		//Utils.ensure(Utils.ensure(capabilities, 'connection')!, 'profiler')!.dynamicRegistration = true;
```

Comment:
Was this here before? I believe this should be `Utils.ensure(capabilities, 'profiler')!.profiler = true;`

(although I'm not exactly sure what that's doing offhand, so someone else more familiar with how this is set up should verify)

Reply:
there's another commented out line like that in another feature above.
The existing line in dataprotocolclient looks like this:
```
public fillClientCapabilities(capabilities: protocol.ClientCapabilities): void {
		ensure(ensure(capabilities, 'connection')!, 'profiler')!.dynamicRegistration = true;
	}
```